compass/ocean/__init__.py

```diff
@@ -3,6 +3,7 @@
 from compass.ocean.tests.global_convergence import GlobalConvergence
 from compass.ocean.tests.global_ocean import GlobalOcean
 from compass.ocean.tests.gotm import Gotm
+from compass.ocean.tests.internal_wave import InternalWave
```

Comment:
I'm much less concerned about the imports but might as well keep them alphabetical, too.